### DISCUSSION TITLE
Greyed out Play button

### DISCUSSION THREAD

**Discussion by 30DH1C0D3R** [kimai/kimai#2753] [Questions & Answers]
Some of our users are seeing a greyed out play button. Instead of it being green and ready to click they see it grey and un-clickable. We have them sign into other profiles on the same machine and it works fine. Anyone else having this issue?

**Comment by kevinpapst** [comment]
I don't even remember that there could be a greyed out button.
Can you share a screenshot?
And please try to upgrade to the latest release first, which might fix the issue for you.

**Comment by 30DH1C0D3R** [comment]
Hi Kevin,

We use a web version of Kimai at the following address: 



And we have 2 people experiencing this issue.

After logging into their Kimai, the Play button is greyed out.


They can still enter times manually but cannot use this feature. I have logged into my login on their machines to see if it was a browser issue, or see if there was some issue with their local machine, but my login on their machines worked perfectly. It has something to do with their specific login.

Let me know if you figure anything out on your end.

Thanks,

Rich
